### ISSUE TITLE
[FR] (Choose BLTOUCH/Probe for Y-Axis)

### ISSUE THREAD

**Issue by Rizzist** [MarlinFirmware/Marlin#19829]
### Description

Basically, I have a belt printer and the Y-Axis is the conventional Z axis in a regular printer. I usedd BLTOUCH on the Y axis for the printer and it didnt work, mainly because of the fact that BLTOUCH is only for Z-AXIS. Support for this would be nice.
### Feature Workflow

1. Turn on Y Axis Probe
2. Use like regular sensor

**Comment by qwewer0** [comment]
Could you describe it a bit more, to what do you mean Y axis is the Z axis in your setup?
Normally the Z axis is for vertical moves, and X/Y axis are for horizontal moves.

**Comment by Rizzist** [comment]
> Could you describe it a bit more, to what do you mean Y axis is the Z axis in your setup?
> Normally the Z axis is for vertical moves, and X/Y axis are for horizontal moves.

Yes, in this case its flipped, where Y axis for vertical moves, X/Z for horizontal moves. Im not sure what else to add to the description other than having another axis have BLTOUCH possible.

**Comment by qwewer0** [comment]
Can't you just switch the Y and Z stepper cables?

Do you have a picture of that kind of printer or a link to a product page?

**Comment by Rizzist** [comment]
> Can't you just switch the Y and Z stepper cables?
> 
> Do you have a picture of that kind of printer or a link to a product page?

The issue with that is because the slicer does not have a way to switch Y and Z axis, its using Blackbelt Cura. I tried vigorously to find a way like writing a python script to replace Y and Z gcode but its not possible unless i use a regular switch instead of probe.

So for temporary fix using the limit switch but in future would love an update as I have the design ready.

**Comment by Rizzist** [comment]
its a homemade belt printer, kind of like blackbelt 3d printer
![image](https://user-images.githubusercontent.com/12385263/96822563-5db6a480-13f8-11eb-988d-0ed626c1c998.png)
X Axis: Left and Right
Y Axis: Up and Down (Slanted)
Z Axis: Forward Backward (Belt)

**Comment by Rizzist** [comment]
I would really like to have this a feature, I got the belt printer working with the regular limit switch, but I have to remove it after leveling and I am constantly afraid that it will crash, and highly prefer the BLTOUCH.

Maybe you can give advice on how to edit marlin to get it working?
